nisystemlink/clients/dataframe/_data_frame_client.py

```diff
@@ -3,11 +3,27 @@
 
 """Implementation of DataFrameClient."""
 
-from systemlink.clients.core._uplink._base_client import BaseClient
+from typing import Optional
+
+from nisystemlink.clients import core
+from nisystemlink.clients.core._uplink._base_client import BaseClient
 from uplink import get, returns
 
 
 class DataFrameClient(BaseClient):
+    def __init__(self, configuration: Optional[core.HttpConfiguration] = None):
+        """Initialize an instance.
+
+        Args:
+            configuration: Defines the web server to connect to and information about
+                how to connect. If not provided, an instance of
+                :class:`core.JupyterHttpConfiguration` is used.
+        """
+        if configuration is None:
+            configuration = core.JupyterHttpConfiguration()
```

Comment:
Should this go in the base client so we have consistent behavior? I imagine we'll also want a fallback where if the environment variables are missing, we look for files (i.e. add Jupyter support to HttpConfigurationManager).